runtime/nls/j9vm/j9vm.nls

```diff
@@ -1661,18 +1661,18 @@ J9NLS_VM_IDLE_TUNING_PLATFORM_OR_GC_POLICY_NOT_SUPPORTED.user_response=Refer to
 
 # first argument is the length of class name
 # second argument is the class name
-J9NLS_VM_CLASS_NOT_VALUE_CAPABLE=The resolved class %2$.*1$s is not value capable
+J9NLS_VM_CLASS_NOT_VALUE_CAPABLE=Unused
```

Comment:
The text should be deleted rather than left blank.

Reply:
I wasn't sure if that was actually supported. If it would cause problems with any of the tooling. Do you know of any messages that have blank text?

Reply:
I found some examples, such as J9NLS_EXELIB_COPYRIGHT_1